core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -107,6 +108,11 @@ class KafkaApisTest {
   private val time = new MockTime
   private val clientId = ""
 
+  @Before
+  def setUp(): Unit = {
```

Comment:
Could we remove this ```@Before``` code? Changing modifier of ```metadataCache``` from "val" to "var" is enough.

Reply:
hey @chia7712 ,

I add this to let each test case to have a clean, uncorrupted `MetadataCache`.  Is there any historical context that we should avoid that cleanup?

Reply:
It seems to me following code is able to generate a clean, uncorrupted ```MetadataCache``` for each test case.

```
var metadataCache = new MetadataCache(brokerId)
```

Also, it is simpler than ```@Before``` block.

Reply:
Hey @chia7712 you're right.  I did some reading and found that I misunderstood JUnit's behavior.  Just updated that part and rebased latest trunk.